Makefile

```diff
@@ -41,16 +51,16 @@ docker-release:
 style:
 	! gofmt -d $$(find . -path ./vendor -prune -o -name '*.go' -print) | grep '^'
 
-test:
-	go get github.com/alicebob/miniredis
-	go test -run '' -o ${GOPATH}/bin/trickster -v
+.PHONY: test
+test: deps
+	go test -o ${GOPATH}/bin/trickster -v ./...
 
-test-cover:
-	go get github.com/alicebob/miniredis
-	go test -run '' -o ${GOPATH}/bin/trickster -coverprofile=cover.out
+.PHONY: test-cover
```

Comment:
Why define some `.PHONY`s above the target and others in a list at the bottom? Should be consistent IMO unless there's a reason.

Reply:
Yea, I was trying to move towards having it be with the targets.

Reply:
Then IMO do it all at once...

Reply:
Done